src/amo/components/SearchResult/index.js

```diff
@@ -73,6 +89,18 @@ export class SearchResultBase extends React.Component<InternalProps> {
     const iconURL = getAddonIconUrl(addon);
 
     let imageURL = iconURL;
+    let addonTitle = <LoadingText />;
+
+    if (addon) {
+      addonTitle = (
+        <Link
```

Comment:
This also needs `stopPropagation()` otherwise the link loads twice and that means you have to press the back button twice to go back.

```diff
diff --git a/src/amo/components/SearchResult/index.js b/src/amo/components/SearchResult/index.js
index 1f1ad5526..bb45d8011 100644
--- a/src/amo/components/SearchResult/index.js
+++ b/src/amo/components/SearchResult/index.js
@@ -96,6 +96,7 @@ export class SearchResultBase extends React.Component<InternalProps> {
         <Link
           to={this.getAddonLink(addon, addonInstallSource)}
           className="SearchResult-link"
+          onClick={(e) => e.stopPropagation()}
         >
           {addon.name}
         </Link>
```

Reply:
Ah, I thought I noticed something odd with the back button while I was testing this, but I wasn't quite sure what it was. Thanks!